fips202/keccakf1600.c

```diff
@@ -24,9 +24,10 @@
 void KeccakF1600_StateExtractBytes(uint64_t *state, unsigned char *data,
                                    unsigned int offset, unsigned int length)
 {
+  unsigned int i;
```

Comment:
For `!SYS_LITTLE_ENDIAN`, there is now a double-declaration of `i`

Reply:
resolved.